### ISSUE TITLE
Restructure Config

### ISSUE THREAD

**Issue by SnakekillerX** [The-Fireplace-Minecraft-Mods/Block-Replacer#24]
Its very difficult to keep track and make edits to your block replacements file with how the current arrays are setup.

Would it be possible to move all of the 10 sections into one larger array?
For example, something similar to this:

    # Use compressed array structure instead of separated arrays (default: false)
    B:compressed=true

    # Enter block data  you would like replaced here
    blockrule <
        *, -2, 256, -1, false, minecraft:netherrack, -1, 1.0, netherex:icy_netherrack, -1
        *, -2, 256, -1, false, minecraft:water, -1, 1.0, netherex:frostburn_ice, -1
        *, -2, 256, -1, false, minecraft:red_mushroom, -1, 1.0, netherex:blue_fire, -1
     >

This would make it much much easier to add/remove/re-organize or even comment out blocks.

**Comment by The-Fireplace** [comment]
I agree that _something_ needs to be done but it'll be hard to do without totally rewriting how the config works. This might just have to be something that has to wait until the 1.15.2 update.
